packages/client/src/components/TextBox/demo.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
@sanjayyepuri we should probably standardize how we are going to do styles, any preference